datafusion/expr/src/utils.rs

```diff
@@ -672,7 +672,7 @@ where
     let mut err = Ok(());
     expr.apply(|expr| {
         if let Err(e) = f(expr) {
-            // save the error for later (it may not be a DataFusionError
+            // Save the error for later (it may not be a DataFusionError
```

Comment:
For docs (`///`) casing is important.
For inline comments, i would be more relaxed (but semantics matters a ton!). But here it would be good to fix lack of closing brace

Reply:
Got it thank you